lib/archive/util_test.go

```diff
@@ -21,6 +21,24 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// INPUT VALIDATION TESTS
+
+func TestNilSpecForAddInclusionExclusion(t *testing.T) {
+	var emptyMounts []string
+	err := AddMountInclusionsExclusions("", nil, emptyMounts, "")
+	assert.Error(t, err)
+
+	emptySpec := FilterSpec{
+		RebasePath: "",
+		StripPath:  "",
+		Exclusions: nil,
+		Inclusions: nil,
+	}
+
+	err = AddMountInclusionsExclusions("", &emptySpec, emptyMounts, "")
```

Comment:
Is there a way to test a no-error case?

Reply:
yes you can do an `assert.NoError(...`

Reply:
But in this case we expect there to be an error. I am unsure of what you are asking here :)